<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="to_5_coins">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TO 5 COINS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="to_1_coins">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>TO 1 COINS</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
